### ISSUE TITLE
Missing device configuration:  Neo Coolcam NAS-PD07Z

### ISSUE THREAD

**Issue by poudenes** [AlCalzone/node-zwave-js#4249]
### Checklist

- [X] It is not in the [configuration DB](https://devices.zwave-js.io/?search=3)

- [X] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)

### Which device is missing?

NAS-PD07Z

### Manufacturer ID

600

### Product Type

16

### Product ID

1824

### Firmware Version

2.6

### Do you have a manual?

No manual digital. Here configuration with all parameters:

<?xml version="1.0" encoding="utf-8"?>
<Product xmlns='https://github.com/OpenZWave/open-zwave' Revision="2">
    <MetaData>
        
        <MetaDataItem name="Name">Neo (Coolcam) NAS-PD07 Motion Sensor (PIR)</MetaDataItem>
        <MetaDataItem name="WakeupDescription">Press and hold the button for 5s until white led lights is on, then release the button before led turn off.</MetaDataItem>
        <MetaDataItem name="InclusionDescription">Set gateway in inclusion mode. Press and hold the button for 5s until white led lights is on, then release the button before led turn off. Blue led will blink with 1s interval until inclusion succesful.</MetaDataItem>
        <MetaDataItem name="ExclusionDescription">Set gateway in exclusion mode. Press and hold the button for 5s until white led lights is on, then release the button before led turn off. Blue led will blink with 0.5s interval until exclusion succesful.</MetaDataItem>
        <MetaDataItem name="ResetDescription">Press and hold button for 10s until pink led lights is on, then release the button before led turn off.</MetaDataItem>
        <ChangeLog>
            <Entry author="David J" date="20 october 2021" revision="1">Initial version, based on PD03Z file</Entry>
        </ChangeLog>
    </MetaData>
    
    
    <CommandClass id="112">
        <Value type="list" genre="config" instance="1" index="1" label="Led indicator enable" value="1" size="1">
            <Help>v2 This parameter is configured the led light on disable or enable.
            </Help>
            <Item label="Disable - Disable led blink. This configuration is not affect inclusion, exclusion and reset." value="0" />
            <Item label="Enable - Enable led blink when device detects a motion event (default)." value="1" />
        </Value>
        
        <Value type="list" genre="config" instance="1" index="2" label="Motion enable" value="1" size="1">
            <Help>This parameter is configured the motion detected if enable or not.
            </Help>
            <Item label="Disable - Motion detected disable." value="0" />
            <Item label="Enable - Motion detected enable (default)." value="1" />
        </Value>
        
        <Value type="list" genre="config" instance="1" index="3" label="Motion alarm once enable" value="1" size="1">
            <Help>This parameter is configured the motion detected event report one time before motion event cleared.
            </Help>
            <Item label="Every time - Motion event alarm will be reported when motion event is detected every time (default)" value="0" />
            <Item label="Only once - Motion event alarm report only once before motion event cleared" value="1" />
        </Value>
        
        <Value type="list" genre="config" instance="1" index="4" label="Luminance associated enable" value="1" size="1">
            <Help>Enables current light intensity to associate the motion event, if there has a motion event detected and the current light intensity is less then the settings is Configuration No. 16, the device will send a BASIC_SET to nodes associated in Group 2. And if the current light intensity is larger then the settings in Configuration No. 16, the device will not send BASIC_SET to nodes associated in Groep 2 (default).
            </Help>
            <Item label="Disable - Light intensity is not associated with motion event" value="0" />
            <Item label="Enable" value="1" />
        </Value>
        
        <Value type="list" genre="config" instance="1" index="5" label="Binary sensor report enable" value="1" size="1">
            <Help>Enables sensor binary report when device detects a motion event.
            </Help>
            <Item label="Disable - Disable sensor binary report when device detects a motion event." value="0" />
            <Item label="Enable  - Enable sensor binary report when device detects a motion event (default)." value="1" />
        </Value>
        
        <Value type="byte" genre="config" instance="1" index="6" label="Motion sensitivity" value="2" min="0" max="15" size="1">
            <Help>This parameter is configured the sensitivity that motion detect. This value (0-15) is larger,the sensitivity is lower, and the distance for motion detection is closer.
            </Help>
        </Value>
        
        <Value type="byte" genre="config" instance="1" index="7" label="Temperature offset value" value="0" min="-120" max="120" size="1">
            <Help>The current temperature measuring value can be add and minus a value (-120-120) by this setting.
                
                Temperature Offset Value = [Value] x 0.1 degree Celsius / Fahrenheit (US).
            </Help>
        </Value>
        
        <Value type="byte" genre="config" instance="1" index="8" label="Humidity offset value" value="0" min="-120" max="120" size="1">
            <Help>The current humidity measuring value can be add and minus a value (-120-120) by this setting.
                
                Humidity Offset Value = [Value] x 0.1 RH%.
            </Help>
        </Value>
        
        <Value type="byte" genre="config" instance="1" index="9" label="Temperature D-value setting" value="10" min="0" max="120" size="1">
            <Help>This configuration sets the changed value (0-120) of the temperature. When the difference from the last report exceeds this setting value, the device will report current temperature value to Z-Wave hubs.
                
                D-value = [Value] x 0.1 degree Celsius / Fahrenheit (US).
            </Help>
        </Value>
        
        <Value type="byte" genre="config" instance="1" index="10" label="Humidity D-value setting" value="20" min="0" max="120" size="1">
            <Help>This configuration sets the changed value (0-120) of the humidity. When the difference from the last report exceeds this setting value, the device will report current humidity value to Z-Wave hubs.
                
                D-value = [Value] x 0.1 RH%.
            </Help>
        </Value>
        
        <Value type="byte" genre="config" instance="1" index="11" label="Luminace D-value setting" units="lux" value="50" min="0" max="120" size="1">
            <Help>This configuration sets the changed value (0-120) of the luminance. When the difference from the last report exceeds this setting value, the device will report current luminance value to Z-Wave hubs.
            </Help>
        </Value>
        
        <Value type="byte" genre="config" instance="1" index="12" label="Basic set level" value="100" min="0" max="100" size="1">
            <Help>This parameter is configured the value (0-100) that BASIC_SET for nodes that associated in Group 2.
                
                100 -	BASIC_SET = 0xFF (ON)
                0 -	BASIC_SET = 0x00 (OFF)
            </Help>
        </Value>
        
        <Value type="byte" genre="config" instance="1" index="13" label="Motion blind time" unit="second" value="8" min="1" max="8" size="1">
            <Help>This configuration is configured the time interval (1-8 seconds) between two motion events triggered, during this period the motion detector will not be triggered even there has someone move in front of motion detector.
            </Help>
        </Value>
        
        <Value type="short" genre="config" instance="1" index="14" label="Basic set off delay time" unit="second" value="30" min="0" max="30000" size="2">
            <Help>This configuration is configured the time delay (0-30000 seconds) for device sending BASIC_SET = 0x00 to nodes that associated in Group 2 when device detects a motion event.
                
                0 -	Not send BASIC_SET = 0x00 command.
                1 ... 30000 -	Time delay count.
            </Help>
        </Value>
        
        <Value type="short" genre="config" instance="1" index="15" label="Motion clear time" unit="second" value="30" min="0" max="30000" size="2">
            <Help>This configuration is configured the time (0-30000 seconds) to clear motion event after a motion event detected. Time to motion clear, the device will send a clear event report to controller.
            </Help>
        </Value>
        
        <Value type="short" genre="config" instance="1" index="16" label="Luminance threshold for associated" unit="lux" value="50" min="0" max="1000" size="2">
            <Help>This configuration is configured the light intensity threshold (0-1000). When ambient intensity is less then this setting , device will consider the current environment is sufficient light. If Configuration No. 3 is set '1' and a motion event is detected, the device will send a BASIC_SET to the nodes which associated in Group 2.
            </Help>
        </Value>
        
        <Value type="short" genre="config" instance="1" index="17" label="Sensor measuring interval" unit="second" value="180" min="0" max="30000" size="2">
            <Help>This configuration is configured the time interval (0-30000 seconds) for light sensor, temperature and humidity sensor measuring. This value is larger, the battery life is longer. And the sensor values changed are not obvious [SIC].
                
                Note that default value is 180 for (battery) Low Power Mode (LPM) and 10 for (DC power) Always On Slave (AOS).
                
                0 -	All sensors are disabled.
            </Help>
        </Value>
        
        <Value type="short" genre="config" instance="1" index="99" label="Light intensity offset calibration" value="5320" min="1" max="32767" size="2">
            <Help>This parameter defines the calibrated scale (1-32767) for ambient light intensity. Because the method and position that the sensor mounted and the cover of sensor will bring measurement error , user can get more real light intensity by this parameter setting. User should run the steps as below for calibrating.
                1 -	Set this parameter value to default (assumes the sensor has been added in a Z-Wave network).
                2 -	Place a digital luxmeter close to sensor and keep the same direction, monitor the light intensity value (Vm) which report to controller and record it. The same time user should record the value (Vs) of luxmeter.
                3 -	The scale calibration formula : k = Vm / Vs
                4 -	The value of k is then multiplied by 1000 and rounded to the nearest whole number.
                5 -	Set the value getting in 5) to this parameter, calibrate finished.
                
                For example, Vm = 300, Vs = 2000, then k = 300 / 2600 = 0.11538
                k = 0.11538 * 1000 = 115.38 - 115
                The parameter should be set to 115.
            </Help>
        </Value>
        
        
    </CommandClass>
    
    <CommandClass id="133">
        <Associations num_groups="2">
            <Group index="1" max_associations="5" label="Lifeline" />
            <Group index="2" max_associations="5" label="Control Commands BASIC_SET" />
        </Associations>
    </CommandClass>
</Product>

**Comment by AlCalzone** [comment]
> ### Manufacturer ID
> 
> 600
> ### Product Type
> 
> 16
> ### Product ID
> 
> 1824
> ### Firmware Version
> 
> 2.6

This does exist: https://devices.zwave-js.io/?jumpTo=0x0258:0x0010:0x0720:0.0 and you should have also found it by searching for `NAS-PD07Z` in the linked configuration DB:
![grafik](https://user-images.githubusercontent.com/17641229/154642685-2c00a47d-72ab-4f1f-9990-3d3d1e5c1d07.png)

**Comment by poudenes** [comment]
Tried everything to found it in the DB but all the time no results... But I see I searched on a other website the Zwave Alliance .... Sorry...

But checked the link you posted. There missing also lots of parameters..... for example:

Humidity D-value setting

    <Value type="byte" genre="config" instance="1" index="10" label="Humidity D-value setting" value="20" min="0" max="120" size="1">
        <Help>This configuration sets the changed value (0-120) of the humidity. When the difference from the last report exceeds this setting value, the device will report current humidity value to Z-Wave hubs.
            
            D-value = [Value] x 0.1 RH%.
        </Help>
    </Value>

**Comment by AlCalzone** [comment]
> There missing also lots of parameters..... for example:

That looks pretty similar to this one, but with a different parameter number (yours seems to be no. 10)
![grafik](https://user-images.githubusercontent.com/17641229/154655231-3cdf63fb-00dc-4822-aad8-46e413dba1de.png)

Note that we take the liberty to rename parameters to be consistent across multiple devices and more easily understood than the (often very terrible) manufacturer-defined labels.

Are you sure that there are actual discrepancies? https://github.com/zwave-js/node-zwave-js/pull/3422 was the last PR that significantly changed the config and it was done to match the parameters that exist on firmware 2.6 (which yours has)

**Comment by poudenes** [comment]
I saw that setting. But if you change some things to make it more standard, that make sense.

**Comment by poudenes** [comment]
I checked some settings and miss one (my idea) a important one:

Manual: 13. Motion Blind Time: The time that motion will be blind for movement.
In Zwave config now number 13 is: Motion Clear time: The time when sensor sends a clear event report
in Zwave Condig number 15 is now Sensor Report interval in manual that is number 17.

My idea is to get it correct:

Parameter 13. Motion Blind Time (settings from 1-8, default 8)
Parameter 15. Motion Clear time
Parameter 17. Sensor Report Interval


```
    <Value type="byte" genre="config" instance="1" index="13" label="Motion blind time" unit="second" value="8" min="1" max="8" size="1">
        <Help>This configuration is configured the time interval (1-8 seconds) between two motion events triggered, during this period the motion detector will not be triggered even there has someone move in front of motion detector.
        </Help>
    </Value>
        
    <Value type="short" genre="config" instance="1" index="15" label="Motion clear time" unit="second" value="30" min="0" max="30000" size="2">
        <Help>This configuration is configured the time (0-30000 seconds) to clear motion event after a motion event detected. Time to motion clear, the device will send a clear event report to controller.
        </Help>
    </Value>
        
    <Value type="short" genre="config" instance="1" index="17" label="Sensor measuring interval" unit="second" value="180" min="0" max="30000" size="2">
        <Help>This configuration is configured the time interval (0-30000 seconds) for light sensor, temperature and humidity sensor measuring. This value is larger, the battery life is longer. And the sensor values changed are not obvious [SIC].
            
            Note that default value is 180 for (battery) Low Power Mode (LPM) and 10 for (DC power) Always On Slave (AOS).
            
            0 -	All sensors are disabled.
        </Help>
    </Value>
```

**Comment by AlCalzone** [comment]
Is is possible that you have the [NAS-PD07ZU1](https://devices.zwave-js.io/?jumpTo=0x0258:0x0020:0x0718:0.0) variant? Or maybe our fingerprints are not correct, they are quite similar:
- 0x0010:0x0720 for PD07Z
- 0x0020:0x0718, 0x0020:0x0720 for PD07ZU1

@luzik any idea?

**Comment by poudenes** [comment]
![image](https://user-images.githubusercontent.com/4399532/154843307-75c20cdd-c814-4c3b-8dfd-e713554ec507.jpeg)

NEO_NAS-PD07Z1

Added the name already in first post...

**Comment by AlCalzone** [comment]
> Added the name already in first post...

In the first post you wrote NAS-PD07Z, which does not match what you write now:
> NEO_NAS-PD07Z1

We have two different versions of the device which are VERY similar:
![grafik](https://user-images.githubusercontent.com/17641229/154843410-d1f542ec-0a5c-4335-b075-5d1b8a4ad0e0.png)

I'm not sure if you got the manufacturer ID etc. from the device itself (in zwavejs2mqtt for example) or somewhere else. The parameters you mention point towards the [second device config](https://devices.zwave-js.io/?jumpTo=0x0258:0x0020:0x0718:0.0), which has the same parameters you mention, but a different ID.

So something is off here, I'm just not sure what.

**Comment by poudenes** [comment]
![image](https://user-images.githubusercontent.com/4399532/154845272-4d53a744-305f-4098-a444-df10750846ee.jpeg)

This is what I have from zwave JS

**Comment by AlCalzone** [comment]
Ok that's a different product ID than in the opening post - and one we don't yet have in our database.

Can you please double-check if these parameters match your device? At first glance it seems so.
https://devices.zwave-js.io/?jumpTo=0x0258:0x0020:0x0718:0.0

**Comment by poudenes** [comment]
https://devices.zwave-js.io/?jumpTo=0x0258:0x0020:0x0718:0.0
Manual have different parameters.

What I can do is scan the manual because its not somewhere on the net.....


Sorry for all the incorrect information. I was on my way and checked it on my phone. Now at home and here the full correct information from Zwave JS:

**Shenzhen Neo Electronics Co., Ltd.
5 in 1 PIR Motion Sensor
NAS-PD07Z**

**Manufacturer Specific v2:**
[120-114-0-manufacturerId] Manufacturer ID: **600**
[120-114-0-productType] Product type: **16**
[120-114-0-productId] Product ID: **1824**

**Version v3:**
[120-134-0-libraryType] Library type: **[3] Enhanced Slave**
[120-134-0-protocolVersion] Z-Wave protocol version: **7.13**
[120-134-0-firmwareVersions] Z-Wave chip firmware versions: **["2.6"]**
[120-134-0-hardwareVersion] Z-Wave chip hardware version: **1**
[120-134-0-sdkVersion] SDK version: **7.13.8**
[120-134-0-applicationFrameworkAPIVersion] Z-Wave application framework API version: **10.13.8**
[120-134-0-applicationFrameworkBuildNumber] Z-Wave application framework API build number: **373**
[120-134-0-hostInterfaceVersion] Serial API version: **unused**
[120-134-0-hostInterfaceBuildNumber] Serial API build number: **0**
[120-134-0-zWaveProtocolVersion] Z-Wave protocol version: **7.13.8**
[120-134-0-zWaveProtocolBuildNumber] Z-Wave protocol build number: **373**
[120-134-0-applicationVersion] Application version: **2.6.0**
[120-134-0-applicationBuildNumber] Application build number: **43707**

**Comment by poudenes** [comment]
Here all parameters that belong to the sensor.
[scan.pdf](https://github.com/zwave-js/node-zwave-js/files/8104229/scan.pdf)

**Comment by luzik** [comment]
Yeah, I can confirm that my NAS-PD07Z have product id 1824. I can also confirm that configuration documentation attached to my device did not match params in device. Configuration provided for this project is mix of documented values and what I found out based on behavior and default values found in configuration.

**Comment by poudenes** [comment]
So, does that mean the config in Zwave is not correct? If you read the manual I send there are some parameters that are important to have and able to change when needed?

**Comment by luzik** [comment]
No, it does mean that changes provided with
https://github.com/zwave-js/node-zwave-js/commit/0e888d4f9b60b8d9e21cc7ff25e22737464b7df5
makes configuration suits much better for my hardware and there is an error in documentation attached to my hardware (or attached documentation is designed for other firmware version or hardware)

**Comment by AlCalzone** [comment]
So if both your devices identify as
[120-114-0-manufacturerId] Manufacturer ID: 600
[120-114-0-productType] Product type: 16
[120-114-0-productId] Product ID: 1824
[120-134-0-firmwareVersions] Z-Wave chip firmware versions: ["2.6"]

And @luzik's actual configuration matches what's in the config file now, either
a) this is correct and should work for @poudenes's device or
b) There are two variants of the device out there which we are not able to distinguish.

@poudenes can you check the config parameters against the actual device if they behave as intended and ignore the attached manual for now?

**Comment by poudenes** [comment]
My config and parameters are the same as the config parameters that has been changed in [0e888d4](https://github.com/zwave-js/node-zwave-js/commit/0e888d4f9b60b8d9e21cc7ff25e22737464b7df5) 

Only 1 important parameter is missing. Manual document this for me on parameter 13. The Motion Blind time.
This will keep motion silence before it will detect again on movement.

Almost all Neo Coolcam motion sensors have this in their config and also in my Zwave JS.

```
    <Value type="byte" genre="config" instance="1" index="13" label="Motion blind time" unit="second" value="8" min="1" max="8" size="1">
        <Help>This configuration is configured the time interval (1-8 seconds) between two motion events triggered, during this period the motion detector will not be triggered even there has someone move in front of motion detector.
        </Help>
    </Value>
```
When you want add this line, the whole numbering has to be change.... 

I guess we have to different types of this sensor/hum

@luzik do you have your manual also scanned? Because then we can compare the manuals that has been send with the actual sensor module...

**Comment by AlCalzone** [comment]
WTF - the param exists, but for the [U1 variant](https://devices.zwave-js.io/?jumpTo=0x0258:0x0020:0x0718:0.0), which has a different product type than yours.

![grafik](https://user-images.githubusercontent.com/17641229/154956247-33d14397-d7ba-4f76-b9e0-d33af2a45bed.png)

> Almost all Neo Coolcam motion sensors have this in their config and also in my Zwave JS.

Are you 100% sure the sensor identifies the same as your other ones who have the parameter? What are their manufacturerId, productType, productId and firmwareVersions?

**Comment by luzik** [comment]
It is quite hard to match documentation with real config params.
After your device factory reset, please compare default param values with default values provided in documentation.

For example:
I found in documentation that the only param with default value 20 is Humidity D-Value. Param number in docs was #10, but when I dump values from hardware, 20 value was under param #9. I experimented with it, and it was really affecting humidity, so I fixed configuration.

My hint is to follow default values to guess real param numbers.

Please also note build number of my firmware, maybe there is a difference.

<img width="1164" alt="Zrzut ekranu 2022-02-21 o 14 01 56" src="https://user-images.githubusercontent.com/6113774/154960355-4868c92e-404a-4751-b940-c35205d5914d.png">

**Comment by poudenes** [comment]
> It is quite hard to match documentation with real config params. After your device factory reset, please compare default param values with default values provided in documentation.
> 
> For example: I found in documentation that the only param with default value 20 is Humidity D-Value. Param number in docs was #10, but when I dump values from hardware, 20 value was under param #9. I experimented with it, and it was really affecting humidity, so I fixed configuration.
> 
> My hint is to follow default values to guess real param numbers.
> 
> Please also note build number of my firmware, maybe there is a difference.
> 
> <img alt="Zrzut ekranu 2022-02-21 o 14 01 56" width="1164" src="https://user-images.githubusercontent.com/6113774/154960355-4868c92e-404a-4751-b940-c35205d5914d.png">

My Version v3 is totally the same!!!
This is my version of ZwaveJS.. maybe there is the problem?
![Schermafbeelding 2022-02-21 om 14 27 07](https://user-images.githubusercontent.com/4399532/154964273-9d14cf66-f859-40fe-8015-c2b45ad88229.png)

**Comment by poudenes** [comment]
I did a DEBUG info and the device have a DB LINK attached:

https://devices.zwave-js.io/?jumpTo=0x0258:0x0010:0x0720:0.0

This model is used for my sensor device

The parameters are different then above given:

https://devices.zwave-js.io/?jumpTo=0x0258:0x0020:0x0718:0.0

**Comment by poudenes** [comment]
Here txt file with full debug information
[debug.txt](https://github.com/zwave-js/node-zwave-js/files/8109752/debug.txt)

**Comment by poudenes** [comment]
> WTF - the param exists, but for the [U1 variant](https://devices.zwave-js.io/?jumpTo=0x0258:0x0020:0x0718:0.0), which has a different product type than yours.
> 
> ![grafik](https://user-images.githubusercontent.com/17641229/154956247-33d14397-d7ba-4f76-b9e0-d33af2a45bed.png)
> 
> > Almost all Neo Coolcam motion sensors have this in their config and also in my Zwave JS.
> 
> Are you 100% sure the sensor identifies the same as your other ones who have the parameter? What are their manufacturerId, productType, productId and firmwareVersions?

Fully sure. The motion sensors (NAS-PD03Z) have a parameter: [106-112-0-6] Re-trigger Interval is same as "motion blind time"

**Comment by luzik** [comment]
@poudenes I assume that you did not change any config values in device registers. As documentation says, there should be one param in config with value 8, documentation also says that this should be under config param #13. Everything you see in  node-zwave-js web front-end is taken from this repo, everything but value. Value is taken from your device register.

Do you get any param value of 8? What is its number?
What value do you have under param #13 ?

**Comment by poudenes** [comment]
@luzik did not change the config values at all.

This is my parameter #13:
**[120-112-0-13] Motion Clear Time**
180 seconds
_(Default: 30)_

I see that latest DB build is 8.11.6 I did a update check but no new updates in my Zwave JS. Im using now 8.11.4
Maybe that is the problem? Not updated to latest  DB file?

**Comment by AlCalzone** [comment]
> Maybe that is the problem? Not updated to latest DB file?

No the file hasn't been touched in a while.

**Comment by poudenes** [comment]
Found this also... 600-1824-16 (0x0258-0x0010-0x0720) I guess these are the hex information 

Manufacturer ID: **600 (0x0258)**
Product type: **16 (0x0010)**
Product ID: **1824 (0x0720)**

**Comment by AlCalzone** [comment]
Yeah, we've already established that your device is identified as this
https://devices.zwave-js.io/?jumpTo=0x0258:0x0010:0x0720:0.0

The question is: what is wrong? The config file, the manual or the identification the device sends?
Or more specifically: what is correct?

Because nothing matches up here:
- 180 seconds for param 13 doesn't match your manual, which states a range of 1...8
- 180 seconds for param 13 doesn't match the default of 30 (if you have really not changed the param)
- The manual (at least partially) matches the U1 variant, but that has a different identification
- And the NAS-PD03Z you mention are a different device entirely.

**Comment by luzik** [comment]
@poudenes, please do factory reset of your device, and dump all configuration values after re-adding.

**Comment by poudenes** [comment]
https://devices.zwave-js.io/?jumpTo=0x0258:0x0010:0x0720:0.0

Param 13 is motion clear time. This stated the default is 30seconds. So my parameters in ZwaveJS are correct. I only changed 30 to 180

This is also what it do. After 180 motion sensor sends a "Clear report"

**Comment by poudenes** [comment]
> @poudenes, please do factory reset of your device, and dump all configuration values after re-adding.

To be clear, Remove the device in ZwaveJS Unpair. Then the procedure to do a factory reset on device itself. Add device again in Zwave JS. Then Debug dump.

**Comment by AlCalzone** [comment]
> So my parameters in ZwaveJS are correct. I only changed 30 to 180

So no problem then? Everything working?
Your train of thought is hard to follow 😅

**Comment by luzik** [comment]
yeah, sounds good, please do it, and we will compare all values you got from device with default values from documentation and our config file

**Comment by poudenes** [comment]
> > So my parameters in ZwaveJS are correct. I only changed 30 to 180
> 
> So no problem then? Everything working? Your train of thought is hard to follow 😅

Sorry... im pinned on the Motion Blind Time... (based on manual)

I will do a reset

**Comment by AlCalzone** [comment]
> im pinned on the Motion Blind Time... (based on manual)

Well, you wrote this:

> Param 13 is motion clear time. This stated the default is 30seconds. So my parameters in ZwaveJS are correct. I only changed 30 to 180
> 
> This is also what it do. After 180 motion sensor sends a "Clear report"

And if that is what the device does, I'd say they gave you the wrong manual.

**Comment by poudenes** [comment]
[debug.txt](https://github.com/zwave-js/node-zwave-js/files/8110009/debug.txt)

Removed, factory reset, add again, get debug, here is the file !

**Comment by AlCalzone** [comment]
I'm not seeing the actual config values in there, can you post a screenshot of the configuration section?

**Comment by poudenes** [comment]
Keep in mind I changed 
[121-112-0-12] Basic Set Off Delay Time from 30 > 180
[121-112-0-13] Motion Clear Time from 30 > 180

30 was for both the default value.

(otherwise my partner gets crazy haha)

![Schermafbeelding 2022-02-21 om 15 54 16](https://user-images.githubusercontent.com/4399532/154979087-1fd065f3-ed36-47ce-867a-3e9825d6989e.png)

**Comment by AlCalzone** [comment]
Ok there are 3 other parameters that are different from the default, but aside from that, this is such a good match that I'm going to say that our config is correct and your device was shipped with the wrong manual.

**Comment by poudenes** [comment]
I guess that is the conclusion... Wrong manual.
